config/fcos/v1_6_exp/validate_test.go

```diff
@@ -234,13 +234,13 @@ func TestValidateGrubUser(t *testing.T) {
 
 func TestValidateConfig(t *testing.T) {
 	tests := []struct {
-		config  Config
-		err     error
+		in      Config
+		out     error
 		errPath path.ContextPath
 	}{
 		// valid config
 		{
-			config: Config{
+			in: Config{
```

Comment:
Good observation! 'in' and 'out' make way more sense!

One last thing if you look at the _TestReportCorrleation_ you can see in the array it did not actually declare the individual fields it was defining.

You don't need to change over to this, Just wanted to make sure that you knew that you can do this.